index.js

```diff
@@ -45,3 +45,9 @@ prevButton.addEventListener("click", () => {
     fetchCharacters();
   }
 });
+
+searchBar.addEventListener("submit", (event) => {
+  event.preventDefault();
+  searchQuery = event.target[0].value;
+  fetchCharacters();
+});
```

Comment:
You need to set the page inside this event listener to `page=1`. Otherwise the pagination does not show the page numbers correctly, when you use the searchbar on another page than page 1.